### ISSUE TITLE
Bug: threadCreate

### ISSUE THREAD

**Issue by exdf0** [aoijs/aoi.js#445]
### Bug Type

Function

### Bugged Function

```js
bot.threadCreateCommand({ //Command
    channel: "947618477613645866", //Channel where its being logged
    code: `channel created ` //Code sent to <channel>
    })
```

### Version of aoi.js

v6.0.3

### Version of Node.js

v16.6.0 (Lowest)

### Relevant log output

```shell
Uncaught TypeError TypeError: Cannot read properties of undefined (reading 'threadCreate')
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by USERSATOSHI** [comment]
Can you update to latest aoi.js version

**Comment by ddodogames** [comment]
it happened to someone else before in the support server, I'm not sure if this occurs on latest version of aoi.js

**Comment by exdf0** [comment]
> Can you update to latest aoi.js version

still same error
Uncaught TypeError TypeError: Cannot read properties of undefined (reading 'threadCreate')

**Comment by USERSATOSHI** [comment]
@Leref is this in your commits

**Comment by ddodogames** [comment]
i assume it's some outdated leftovers of v5 unupdated stuff. (Corrected)

**Comment by exdf0** [comment]
any news?

**Comment by USERSATOSHI** [comment]
well no news right now as leref is offline but this will fixed in next version

**Comment by exdf0** [comment]
so ?

**Comment by Leref** [comment]
This will be solved in next update.

**Comment by exdf0** [comment]
is it fixed on 6.2.0 ?

**Comment by Leref** [comment]
Resolved in c84e5b1.
